.github/workflows/test.yml

```diff
@@ -77,7 +89,11 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
       - uses: taiki-e/install-action@cargo-hack
       - uses: Swatinem/rust-cache@v2
-      - run: cargo hack check --feature-powerset --optional-deps serde,rkyv --skip default --skip __internal_bench --skip __doctest --skip iana-time-zone --skip pure-rust-locales
+      - run: |
+          cargo hack check --feature-powerset --optional-deps serde,rkyv \
+            --skip default --skip __internal_bench --skip __doctest \
+            --skip iana-time-zone --skip pure-rust-locales
+        shell: bash
```

Comment:
tl;dr needed because of "line continution" marks.

So by default, the Windows shell is powershell. Powershell uses `` ` `` as line continuation marks. So I forced Windows to use `bash` (github Windows images have bash installed) which uses line continuation mark `\` .

The other way to do this is to have two `run` statements; a `run` statement that matches `matrix.os` on Windows and that command script uses line continuation `` ` ``, then the same `run` statement matches `matrix.os` on !Windows (and thus be running bash) and use line continuation mark `\`.  But that's kinda' messy.

Reply:
Okay, can you add a comment about this on the same line?

Reply:
Commented.